### DISCUSSION TITLE
ConfirmPopup Component add appendTo property

### DISCUSSION THREAD

**Discussion by ilFortu** [primefaces/community#1807] [PrimeNG]
Is it possible to add `appendTo` parameter like for other components?
My problem is that I need to render the HTML not under the body but in an inner element

**Comment by cetincakiroglu** [comment]
Hi,

There isn't such a feature in confirm popup, confirm dialog might help?https://primeng.org/confirmdialog

**Reply by ilFortu** [reply]
Confirm Dialog is fine, but I would like to use ConfirmPopup, I also would need the `appendTo` behavior, is not possible to implement it?

**Comment by JacobF7** [comment]
@cetincakiroglu - we are currently experiencing the same issue in a microfrontend environment where we are unable to append the popup to the our current application and instead it's being appended to the Body. Any reason why we are not using the same approach taken in Prime React [here](https://primereact.org/confirmpopup/#api.ConfirmPopup.props.appendTo)?
